hedera-mirror-test/k6/src/lib/parameters.js

```diff
@@ -148,6 +149,20 @@ export const computeAccountParameters = (configuration) =>
     });
   });
 
+export const computeBlockParameters = (configuration) =>
+  computeProperties(['DEFAULT_BLOCK_NUMBER', 'DEFAULT_BLOCK_HASH'], () => {
+    const extractProperties = (block) => {
+      return {
+        DEFAULT_BLOCK_NUMBER: block.number,
+        DEFAULT_BLOCK_HASH: block.hash,
+      };
+    };
+    return getPropertiesForEntity(configuration, extractProperties, {
+      entitiesKey: blockListName,
+      entityIdResponseKey: 'blocks',
```

Comment:
```suggestion
      queryParamMap: {limit: 1},
```
Override limit to 1 since we just need one block.

`entityIdResponseKey` won't work for blocks endpoint because the response and query parameters are structured differently from e.g. `/accounts`. `getPropertiesForEntity` will still work just because only 1 block is needed.